attestation/src/extensions.rs

```diff
@@ -0,0 +1,47 @@
+//! Standardized X.509 Certificate Extensions
```

Comment:
Copyright headers for this and other new files?

Reply:
Fixed as well.